### PR TITLE
feat: add severity + add tests

### PR DESCRIPTION
**Pull request by StephaneRob**
## 📖 Description and reason

- Add severity to mix audit report
- Add test on formatting

### CODE REVIEW

File: lib/mix_audit/formatting/human.ex [hunk 1]
```diff
@@ -23,6 +23,7 @@ defmodule MixAudit.Formatting.Human do
     #{colorized_text("Title:", :red)} #{String.trim(vulnerability.advisory.title)}
     #{colorized_text("Vulnerable versions:", :red)} #{versions(vulnerability.advisory.vulnerable_version_ranges)}
     #{colorized_text("First patched versions:", :red)} #{versions(vulnerability.advisory.first_patched_versions)}
+    #{colorized_text("Severity:", :red)} #{vulnerability.advisory.severity}
```
Comment (remi) [inline lib/mix_audit/formatting/human.ex:26]: I think we should keep the version information as lowest as possible.

```suggestion
    #{colorized_text("Severity:", :red)} #{vulnerability.advisory.severity}
    #{colorized_text("Vulnerable versions:", :red)} #{versions(vulnerability.advisory.vulnerable_version_ranges)}
    #{colorized_text("First patched versions:", :red)} #{versions(vulnerability.advisory.first_patched_versions)}
```

**Review by remi — CHANGES_REQUESTED** [review]
You’re on 🔥 @StephaneRob! Just a little thing to change and I think it’s ready to merge!

**Comment by StephaneRob** [comment]
@remi I've changed position of severity info.

**Comment by remi** [comment]
Awesome!

**Comment by remi** [comment]
@StephaneRob v2.0.1 will include this new feature!

**Comment by StephaneRob** [comment]
🔥 thanks @remi!

**Pull request merged**